_posts/2022-04-28-introducing-uswds-3-0.md

```diff
@@ -58,14 +58,14 @@ IE11 has fallen below our 2% support threshold as monitored by analytics.usa.gov
 
 [USWDS browser support (2% rule)](https://designsystem.digital.gov/documentation/developers/#browser-support) →
 
-[See current analytics from analytics.usa.gov](https://analytics.usa.gov/) → 
+[See current analytics from analytics.usa.gov](https://analytics.usa.gov/)
 
 ### Support some Javascript-required functionality 
 
 We believe that JavaScript-disabled environments fall below our 2% support threshold, and that we may require JavaScript for some design system functionality. While we have not made any changes to our codebase that affect how we use JavaScript with our components, we may make changes that require JavaScript for some functionality as we move forward. When we do make any changes like this, we'll be sure to note it in our release notes.
 
 ### Support USWDS 2 with maintenance and security patches for one year
 
-In addition to supporting active development on USWDS 3, We will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
+In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).
 
 [Long-term support of version 2.x]({{ site.baseurl }}/documentation/developers/#long-term-support-of-v2x) → 
```

Comment:
@thisisdano
I am not able to find the content for this link

Reply:
It should be there — possibly related to caching of the old README?

<img width="775" alt="Screen Shot 2022-04-27 at 3 38 39 PM" src="https://user-images.githubusercontent.com/11464021/165642885-f9e15087-e411-4441-98ba-08569654a76e.png">

Reply:
My bad. I cleared the ole jekyll cache and all was well. Marking this as resolved.